index.html

```diff
@@ -130,6 +130,8 @@ <h2 class="date"></h2>
     <p><b>Source</b>: Coronavirus (COVID-19) Global Cases by the Center for Systems Science and Engineering (CSSE) at Johns Hopkins University (JHU)</p>
     <p><b>Hyperlink</b>: <a href="arcgis.com/apps/opsdashboard/index.html#/bda7594740fd40299423467b48e9ecf6">arcgis.com/apps/opsdashboard/index.html#/bda7594740fd40299423467b48e9ecf6</a></p>
     <script src="viz.js"></script>
+    <br>
+ <p>According to the <b>World Health Organization</b> (<b>WHO</b>), <i>confirmed case</i> is defined as, “a person with laboratory confirmation of COVID-19 [coronavirus disease 2019] infection, irrespective of clinical signs and symptoms.”
```

Comment:
definition preceded by line break

Reply:
Should we distinguish between "coronavirus" and "COVID-19?"

Reply:
COVID-19 is the disease, whereas we care about the virus.